pontoon/base/templates/add_project.html

```diff
@@ -0,0 +1,21 @@
+{% extends "base.html" %}
+
+{% block title %}{{ title }}{% endblock %}
+
+{% block middle %}
+<section id="error">
+  <div class="inner">
+    <div id="box">
+      <a href={{ url('pontoon.admin.project.new') }}>
+        <div class="fas fa-plus-square"></div>
+      </a>
+      <p>There aren't any project set up currently</p>
+    </div>
+    <div id="subtitle">
+      <a href={{ url('pontoon.admin.project.new') }}>
+        {% block description %}Set up your first project{% endblock %}
```

Comment:
You don't need the `description` block here, and to avoid adding whitespace, you should just put the entire `<a>` node in one line.